### PR TITLE
Zendesk conf

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: k8s/regions/oregon-a/stage.yaml [hunk 1]
```diff
@@ -135,3 +135,7 @@ app:
     zendesk_api_token: SECRET
     zendesk_user_email: SECRET
     zendesk_subdomain: SECRET
+    zendesk_category_field_id: 360047206172
+    zendesk_os_field_id: 360018604871
+    zendesk_vpn_field_id: 360047198211
+    zendesk_ticket_form_id: 360000417171
```
Comment (LeoMcA) [inline k8s/regions/oregon-a/stage.yaml:141]: no newline

File: k8s/regions/oregon-b/stage.yaml [hunk 2]
```diff
@@ -134,3 +134,7 @@ app:
     zendesk_api_token: SECRET
     zendesk_user_email: SECRET
     zendesk_subdomain: SECRET
+    zendesk_category_field_id: 360047206172
+    zendesk_os_field_id: 360018604871
+    zendesk_vpn_field_id: 360047198211
+    zendesk_ticket_form_id: 360000417171
```
Comment (LeoMcA) [inline k8s/regions/oregon-b/stage.yaml:140]: Only the `ticket_form_id` ought to change between stage/prod, so can we not set the `*_field_id` variables - and pick up the defaults from settings.py - until we definitely need them?

File: kitsune/customercare/zendesk.py [hunk 3]
```diff
@@ -94,11 +87,11 @@ def create_ticket(
         ticket = Ticket(
             subject=subject,
             comment={"body": description},
-            ticket_form_id=TICKET_FORM_ID,
+            ticket_form_id=settings.ZENDESK_TICKET_FORM_ID,
             custom_fields=[
-                {"id": PRODUCT_FIELD_ID, "value": product},
-                {"id": CATEGORY_FIELD_ID, "value": category},
-                {"id": OS_FIELD_ID, "value": os},
+                {"id": settings.ZENDESK_VPN_FIELD_ID, "value": product},
```
Comment (LeoMcA) [inline kitsune/customercare/zendesk.py:92]: this should be called `ZENDESK_PRODUCT_FIELD_ID`

File: kitsune/settings.py [hunk 4]
```diff
@@ -1205,3 +1205,8 @@ def filter_exceptions(event, hint):
 ZENDESK_SUBDOMAIN = config("ZENDESK_SUBDOMAIN", default="")
 ZENDESK_API_TOKEN = config("ZENDESK_API_TOKEN", default="")
 ZENDESK_USER_EMAIL = config("ZENDESK_USER_EMAIL", default="")
+# TODO: switch to prod when available
+ZENDESK_VPN_FIELD_ID = config("ZENDESK_VPN_FIELD_ID", default="360047198211", cast=int)
+ZENDESK_OS_FIELD_ID = config("ZENDESK_OS_FIELD_ID", default="360018604871", cast=int)
+ZENDESK_CATEGORY_FIELD_ID = config("ZENDESK_CATEGORY_FIELD_ID", default="360047206172", cast=int)
```
Comment (LeoMcA) [inline kitsune/settings.py:1211]: nit: can we swap these so they're in the same order as they appear on the form?

**Review by LeoMcA — APPROVED** [review]
r+wc

File: kitsune/customercare/zendesk.py (hunk 3 above)
Comment (akatsoulas) [reply kitsune/customercare/zendesk.py:92]: what if we add another product though?

File: k8s/regions/oregon-b/stage.yaml (hunk 2 above)
Comment (akatsoulas) [reply k8s/regions/oregon-b/stage.yaml:140]: I will go with better explicit that implicit

Comment (LeoMcA) [reply k8s/regions/oregon-b/stage.yaml:140]: With such similar IDs across all the fields (it's hard to glance at them and immediately notice the difference) I think being explicit just makes it more confusing. Could these at least be ordered in the same way as the fields in the form/settings to make comparing them top-to-bottom easier.

File: kitsune/customercare/zendesk.py (hunk 3 above)
Comment (LeoMcA) [reply kitsune/customercare/zendesk.py:92]: this is the ID for the product field, not VPN specifically - another product would use the same field, with a different value

**Pull request merged**